Source/GameBaseFramework/Private/Tasks/GBFWaitForTriggerManagerEvent.cpp

```diff
@@ -0,0 +1,69 @@
+﻿#include "Tasks/GBFWaitForTriggerManagerEvent.h"
+
+#include "Gameplay/Components/GBFTriggerManagerComponent.h"
+
+UGBFWaitForTriggerManagerEvent * UGBFWaitForTriggerManagerEvent::WaitForTriggerManagerEvent( UObject * world_context_object,
+    UGBFTriggerManagerComponent * trigger_manager_component,
+    const bool broadcast_trigger_count_on_activate,
+    const bool trigger_once )
+{
+    auto * my_obj = NewObject< UGBFWaitForTriggerManagerEvent >();
+    my_obj->TriggerManagerComponent = trigger_manager_component;
+    my_obj->bBroadcastTriggerCountOnActivate = broadcast_trigger_count_on_activate;
+    my_obj->bTriggerOnce = trigger_once;
+    return my_obj;
+}
+
+void UGBFWaitForTriggerManagerEvent::Activate()
+{
+    Super::Activate();
+
+    if ( TriggerManagerComponent != nullptr )

```

Comment:
maybe end the task if the component is null?

Reply:
Done.